src/main.go

```diff
@@ -22,7 +22,7 @@ func NewTemporalCloudProvider(temporalCloudAdapter ITemporalCloudAdapter, config
 
 	err := temporalCloudAdapter.Flags().Parse(os.Args)
 	if err != nil {
-		return nil, fmt.Errorf("unable to parse flags: %v", err)
+		return nil, fmt.Errorf("failed to parse flags: %v", err)
 	}
```

Comment:
The error message was changed from 'unable to parse flags' to 'failed to parse flags'. This change improves the uniformity of error messages throughout the function, as all other error messages begin with 'failed to...'. This consistency can help with debugging and log analysis.